druid/src/widget/slider.rs

```diff
@@ -120,6 +120,20 @@ impl Slider {
         value
     }
 
+    fn calculate_scroll_value(&self, mouse_delta: f64) -> f64 {
+        let increment = if let Some(step) = self.step {
+            step
+        } else {
+            0.1
```

Comment:
In some cases the difference between min and max is smaller than 0.1. This value should depend on min and max.